docs/docs-components/SandpackExample.js

```diff
@@ -59,9 +51,9 @@ function SandpackContainer({
             <SandpackCodeEditor
               wrapContent
               style={{
-                height: height > MIN_HEIGHT ? height : MIN_HEIGHT,
+                height: height != null && height > MIN_EDITOR_HEIGHT ? height : MIN_EDITOR_HEIGHT,
                 flex: layout === 'column' ? 'none' : null,
-                order: 1,
+                order: layout === 'column' ? 1 : null,
```

Comment:
why are we undoing this? 


I was moving the code editor below de example for consistency
<img width="838" alt="Screen Shot 2022-08-04 at 9 22 32 PM" src="https://user-images.githubusercontent.com/10593890/182935717-9490d6b7-c2e4-4c19-a93d-3bc3a29c3e6b.png">

Reply:
@AlbertCarreras I did this since CodeSandbox also always has the editor on the left. So it makes it more consistent when users click `Open in CodeSandbox`.

Otherwise the code "jumps" from 1 side to the next:
![image](https://user-images.githubusercontent.com/127199/183237136-2762a042-6a8b-4d1a-aa46-798c3e2aaefc.png)

![image](https://user-images.githubusercontent.com/127199/183237156-a73ae1e5-1b15-44be-9498-5b3ba7c524b6.png)

For now I'll revert this change though, to get this diff landed